### ISSUE TITLE
Demo not working

### ISSUE THREAD

**Issue by ovidius72** [charlieduong94/marko-path-router#1]
Maybe it's something related to my configuration but i'm not able to run the demo. 
After yarn install, I run the ./run-demo.sh script and open the browser at localhost:8080 but from the browser console I see the following error:

`input props Object {routes: Array[3]}`
`index.js:39 initial Route Object {path: "/", initial: true}`
`index-browser.js:37 Uncaught Error: Unable to find route /
    at History.push (index-browser.js:37)
    at Widget.init (index.js:41)
    at initWidget (init-widgets-browser.js:195)
    at initClientRendered (init-widgets-browser.js:274)
    at Object.initClientRendered (init-widgets-browser.js:271)
    at WidgetsContext.initWidgets (WidgetsContext.js:97)
    at EventEmitter.<anonymous> (index-browser.js:71)
    at EventEmitter.emit (events.js:81)
    at RenderResult.afterInsert (RenderResult-browser.js:68)
    at RenderResult.replace (RenderResult-browser.js:83)
`

Could it be any dependency is missing ?

**Comment by charlieduong94** [comment]
Hello @ovidius72, sorry, but at the moment, this module is completely not ready for use. At the time when I started working on this, I was trying to experiment a bit with how routing could work with marko v3 but never got around to completing it.

I do want to eventually get back to this (this time around with marko v4). For now, I'll add a README to make it clear that this module isn't ready and if you would like I could notify you when it is in a decent state to play with.

**Comment by ovidius72** [comment]
Thanks a lot @charlieduong94. It would be great to have routings in marko. It's just a few days I'm trying to switch from angular/react to marko and figuring out how to implement routings, so sure, if you are planning to get back on it I'm pretty sure a lot of people would appreciate. Thank you again.

**Comment by charlieduong94** [comment]
Hey @ovidius72, so I finally got around to working on this module again. Sorry for the long delay, you probably have found an alternative routing solution by now.

 I do have this in a working state now, although there is definitely room for improvement. I've updated the docs to explain how to set up the router.

I moved the demo out into a separate repo:
https://github.com/charlieduong94/marko-path-router-demo 
(it's a little bare right now, but more will be added soon)

If you are still interested in this, check out the demo whenever you have time. If you run into any issues, have questions, or have any ideas with how this can be improved, feel free to open up an issue.
